### PR TITLE
NO-ISSUE: resolved dependency on ocm in common

### PR DESCRIPTION
**Pull request by batzionb**

### CODE REVIEW

File: src/common/components/featureSupportLevels/utils.ts [hunk 1]
```diff
@@ -47,11 +43,7 @@ export const getLimitedFeatureSupportLevels = (
     }
     return ret;
   } catch (err) {
-    captureException(
-      err,
-      `Failed to get cluster ${cluster.id} feature support levels`,
-      Sentry.Severity.Warning,
-    );
+    console.error(`Failed to get cluster ${cluster.id} feature support levels`);
```
Comment (jkilzi) [inline src/common/components/featureSupportLevels/utils.ts:46]: Idea: we could pass the `captureException` as an additional arg to this function. Later any consumer can decide "how to capture the exception". In OCM we report it to Sentry in a disconnected environment they might decide to do something else that doesn't require contacting an external service. WDYT?

**Review by jkilzi — COMMENTED** [review]
I left some ideas, let me know what you think...

File: src/common/components/featureSupportLevels/utils.ts (hunk 1 above)
Comment (batzionb) [reply src/common/components/featureSupportLevels/utils.ts:46]: this function is called from a component in common, that's called from another component in common, etc. I think the only way to solve this is with the errorhandler context

Comment (jkilzi) [reply src/common/components/featureSupportLevels/utils.ts:46]: That's not correct
image

**Comment by batzionb** [comment]
opened a [new PR](https://github.com/openshift-assisted/assisted-ui-lib/pull/1098) instead of this one

**Pull request closed without merging**